grakn-dashboard/src/js/User.js

```diff
@@ -62,6 +62,20 @@ export default {
     return (modalShown === 'true');
   },
 
+    // ------ Commands modal setter and getter (has it already be shown to the user) ----//
+    setCmdModalShown(shown) {
+      localStorage.setItem('commands_modal', shown);
+    },
+  
+    getCmdModalShown() {
+      const cmdModalShown = localStorage.getItem('commands_modal');
+      if (!cmdModalShown) {
+        this.setCmdModalShown(false);
+        return false;
+      }
+      return (cmdModalShown === 'true');
+    },
+
```

Comment:
Why do we need to remember if the user has already seen the command modal?

Reply:
We do that only for the join community because we want to show that modal immediately to the user the first time the user open dashboard, without waiting for him to click on the link. So we don't need it for commands, we should show this modal only if the user clicks on it.